contrib/hawq-docker/Makefile

```diff
@@ -0,0 +1,205 @@
+#!/usr/bin/make all
+
+THIS_MAKEFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
+TOP_DIR := $(abspath $(dir ${THIS_MAKEFILE_PATH}))
+NDATANODES := 3
+CUR_DATANODE := 1
```

Comment:
@stanlyxiang is right about NDATANODES and CUR_DATANODE.

Good suggestion on configuring the number. For now user can define the value via parameters to 'make', for example
 $ make NDATANODES=6